addons/hr_holidays/security/ir_rule.xml

```diff
@@ -4,7 +4,7 @@
     <record id="property_rule_holidays_employee" model="ir.rule">
         <field name="name">Employee Holidays</field>
         <field name="model_id" ref="model_hr_holidays"/>
-        <field name="domain_force">[('employee_id.user_id','=',user.id)]</field>
+        <field name="domain_force">['|', ('employee_id.user_id','=',user.id), ('state', '=', 'validate')]</field>
```

Comment:
So we agree to remove it in this task. 
https://www.odoo.com/web#id=21650&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720